engine/controllers/server.cc

```diff
@@ -352,6 +352,11 @@ void server::LoadModel(const HttpRequestPtr& req,
 
   LOG_TRACE << "Load model";
   auto& en = std::get<EngineI*>(engines_[engine_type].engine);
+  if (engine_type == kLlamaEngine) {  //fix for llamacpp engine first
+    auto config = file_manager_utils::GetCortexConfig();
+    en->SetFileLogger(config.maxLogLines, config.logFolderPath + "/" +
```

Comment:
nit, IMO using `std::filesystem::path` is safer compare to string concatenation

Reply:
I fixed it in this pr #1200